internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -77,6 +77,9 @@ type RegistryPod struct { //nolint:maligned
 	// The secret's key for this file must be "cert.pem".
 	CASecretName string
 
+	// SkipTLS controls whether to ignore SSL errors while pulling bundle image from registry server.
```

Comment:
```suggestion
```

Since this is not being used in registry pod creation anymore.

Reply:
removed this comment.